ironfish/src/rpc/adapters/ipcAdapter.test.ts

```diff
@@ -26,6 +26,9 @@ describe('IpcAdapter', () => {
         enableRpc: false,
         enableRpcIpc: false,
       },
+      internalConfigOverrides: {
+        rpcAuthToken: 'test token',
```

Comment:
we don't need this since IPC has no auth

Reply:
removed